requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+-r requirements.txt
```

Comment:
will this file be used when you execute `python setup.py develop`? Or you have to make it explicit somehow?